controller/src/main/java/org/jboss/as/controller/operations/global/ReadFeatureDescriptionHandler.java

```diff
@@ -122,16 +122,16 @@
 
     private final ImmutableCapabilityRegistry capabilityRegistry;
 
-    private static final String $PROFILE_PREFIX = "$profile.";
-    private static final String ADDRESS_PARAMETERS = "addr-params";
-    private static final String ADDRESS_PARAMETERS_MAPPING = "addr-params-mapping";
-    private static final String DOMAIN_EXTENSION = "domain.extension";
-    private static final String FEATURE_ID = "feature-id";
-    private static final String HOST_EXTENSION = "host.extension";
-    private static final String OPERATION_PARAMETERS = "op-params";
-    private static final String OPERATION_PARAMETERS_MAPPING = "op-params-mapping";
-    private static final String PARAMETERS = "params";
-    private static final String REFERENCES = "refs";
+    public static final String $PROFILE_PREFIX = "$profile.";
+    public static final String ADDRESS_PARAMETERS = "addr-params";
+    public static final String ADDRESS_PARAMETERS_MAPPING = "addr-params-mapping";
+    public static final String DOMAIN_EXTENSION = "domain.extension";
+    public static final String FEATURE_ID = "feature-id";
+    public static final String HOST_EXTENSION = "host.extension";
+    public static final String OPERATION_PARAMETERS = "op-params";
+    public static final String OPERATION_PARAMETERS_MAPPING = "op-params-mapping";
+    public static final String PARAMETERS = "params";
+    public static final String REFERENCES = "refs";
```

Comment:
All these constants should remain private to the ReadFeatureDescriptionHandler.java class.
If some needs to be referenced from the test (e.g. `ADDRESS_PARAMETERS`), they should be moved to `org.jboss.as.controller.descriptions.ModelDescriptionConstants`.